joomla/joomla-cms
# Changed the "Start Update" button state on the basis of checkbox value

### Summary of Changes

Changed button state on the basis of checkbox value.

### Testing Instructions

1. System-->Update-->Joomla.
2. Check that the "Start button" is disabled by default after applying the patch.
3. Click on the checkbox and make sure that the "Start button" is getting enabled/disabled on the basis of checkbox value.



### Actual result BEFORE applying this Pull Request

"Start Button" is always active irrespective of the checkbox value.


### Expected result AFTER applying this Pull Request

"Start Button" gets enabled/disabled on the basis of checkbox value.
